### DISCUSSION TITLE
NUnit4 support

### DISCUSSION THREAD

**Discussion by louisnayegon** [reqnroll/Reqnroll#182] [Q&A]
I have noticed in your test code you are still using version 3 of NUnit
https://github.com/reqnroll/Reqnroll/blob/574d7384c4ea07674405c4972ff0b39e55f01638/Plugins/Reqnroll.NUnit.ReqnrollPlugin/Reqnroll.NUnit.ReqnrollPlugin.csproj#L10

Does this work with v4?
If not when do you expect to support this?

**Comment by ajeckmans** [comment]
We just came out of a period where updates were lacking. If it is possible to support v3 and v4 then we can easily do this, and we would welcome any PR for it.

**Comment by gasparnagy** [comment]
> Does this work with v4?

@louisnayegon It is also a help for us if you try and see if you encounter any problems.
